ddl/callback_test.go

```diff
@@ -20,16 +20,17 @@ import (
 	goctx "golang.org/x/net/context"
 )
 
-type testDDLCallback struct {
+type TestDDLCallback struct {
 	*BaseCallback
 
-	onJobRunBefore func(*model.Job)
-	onJobUpdated   func(*model.Job)
-	onBgJobUpdated func(*model.Job)
-	onWatched      func(ctx goctx.Context)
+	onJobRunBefore       func(*model.Job)
+	onJobUpdated         func(*model.Job)
+	OnJobUpdatedExported func(*model.Job)
```

Comment:
In my opine, the only difference between `OnJobUpdatedExported` and `OnJobUpdated` is if we have not null `OnJobUpdatedExported`, it will shade `OnJobUpdated`. And it's only used in `ddl/ddl_db_change_test.go`. So what we need this?